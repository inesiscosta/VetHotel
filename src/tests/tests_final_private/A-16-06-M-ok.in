7
2
vvvv1
nome nove
VETTTT
ZKPRZKP
VE
V
TR
ZKPT
VET
1
0
0
