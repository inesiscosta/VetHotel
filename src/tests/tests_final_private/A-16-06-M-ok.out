Main Menu
1 - New File
2 - Open File
3 - Save File
4 - Next Season
5 - Calculate Global Satisfaction
6 - Manage Animals
7 - Manage Employees
8 - Manage Habitats
9 - Manage Vaccines
10 - Search
0 - Exit
Choose an option: Manage Employees
1 - List all Employees
2 - Register Employee
3 - Add Responsibility
4 - Remove Responsibility
5 - Calculate Employee Satisfaction
0 - Exit
Choose an option: Employee ID: Employee Name: Type of Employee? (VET or ZKP) Type of Employee? (VET or ZKP) Type of Employee? (VET or ZKP) Type of Employee? (VET or ZKP) Type of Employee? (VET or ZKP) Type of Employee? (VET or ZKP) Type of Employee? (VET or ZKP) Manage Employees
1 - List all Employees
2 - Register Employee
3 - Add Responsibility
4 - Remove Responsibility
5 - Calculate Employee Satisfaction
0 - Exit
Choose an option: ZKP|T1|ZKP one
VET|V1|Vet one
VET|vvvv1|nome nove
Manage Employees
1 - List all Employees
2 - Register Employee
3 - Add Responsibility
4 - Remove Responsibility
5 - Calculate Employee Satisfaction
0 - Exit
Choose an option: Main Menu
1 - New File
2 - Open File
3 - Save File
4 - Next Season
5 - Calculate Global Satisfaction
6 - Manage Animals
7 - Manage Employees
8 - Manage Habitats
9 - Manage Vaccines
10 - Search
0 - Exit
Choose an option: 
